ArduPlane/geofence.pde

```diff
@@ -452,6 +469,10 @@ static bool geofence_set_enabled(bool enable, GeofenceEnableReason r) {
     return false;
 }
 
+static bool geofence_set_floor_enabled(bool floor_enable) {
+    return false;
```

Comment:
That function allows the code to compile when the fence isn't present.  See all the functions around it.